templates/parachain/runtime/src/lib.rs

```diff
@@ -414,6 +414,7 @@ impl pallet_message_queue::Config for Runtime {
 	type HeapSize = sp_core::ConstU32<{ 64 * 1024 }>;
 	type MaxStale = sp_core::ConstU32<8>;
 	type ServiceWeight = MessageQueueServiceWeight;
+	type IdleMaxServiceWeight = MessageQueueServiceWeight;
```

Comment:
I think this could break current integration tests, would be safer to set it to `()` everywhere.

Reply:
Do we need this for westend and system chains?

Reply:
We could test it on those, yea. Please enable on Rococo/Westend+System chains.